modules/services/lorri.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lorri;
+in
+{
+  options =  {
+    services.lorri = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the lorri service.";
+      };
+      
+      logFile = mkOption {
+        type = types.nullOr types.path;
+        default = "/var/tmp/lorri.log";
```

Comment:
What would be a good default value here?
`/var/log/lorri.log`, `/Library/Logs/lorri.log` and `/run/current-system/user/Library/lorri.log` all give permission errors. What about a log directory under `/run/current-system/user/Logs`?

Reply:
Using `StandardOutPath` doesn't do any log rotation either so it's best not to enable that by default. As for the location, yeah there's not really a great location that doesn't require extra setup to get the permissions right first. Ideally would do that part automatically for agents with logging.

Reply:
Ok, I disable StandardOutPath and StandardErrorPath by default.